postgraphile/postgraphile/__tests__/queries/v4/sub.json5

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    data: {
+      listen: {
+        __typename: "ListenPayload",
+        relatedNodeId: "WyJYIiwxXQ==",
+        relatedNode: null,
```

Comment:
I believe these show the current issue, not sure how to write an expect for this in the test yet...